pyproject.toml

```diff
@@ -1,3 +1,48 @@
+[tool.coverage.run]
+branch = true
+source = ["OpenSSL", "tests/"]
+
+[tool.coverage.paths]
+source = [
+    "src/OpenSSL",
+    ".tox/*/lib/python*/site-packages/OpenSSL",
+    ".tox/pypy/site-packages/OpenSSL",
+]
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.mypy]
+warn_unused_configs = true
+follow_imports = "skip"
+strict = true
+exclude = ['SSL\.py$']
```

Comment:
Nope, should be fine without: `(?x)` enables `re.VERBOSE`, which doesn't have any effect here (since there's no comments or significant whitespace in the regexp itself).